templates/apollo-starter-ts/README.md

```diff
@@ -7,3 +7,25 @@ Starter template using graphback and apollo-server
 docker-compose up -d
 npm start
 ```
+
+### Project structure
+
+```
+|-----generated
+       |-----schema.graphql
+       |-----resolvers.ts
+|-----model
+       |-----<your-model-name>.graphql
+|-----src
```

Comment:
i will just explain root. rest will change so often

Reply:
Just the directory structure, not the content inside